students/khtruong/lesson_03/list_lab.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+"""Sample module docstrings text
+"""
+
+
+def fruit_list():
+    '''Return fruit list.'''
+    return ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+
+def add_fruit():
+    '''Return user input for type of fruit to add.'''
+    return input('Enter a type of fruit to add to list> ')
+
+
+def remove_fruit(l):
+    '''Return user input for type of fruit to delete.'''
+    keepgoing = True
+    while keepgoing:
+        del_item = input('Enter a type of fruit to remove from list> ')
+        # looking for valid item...keeps asking user until do
+        for item in l:
+            if item == del_item:
+                keepgoing = False
+                break
+        else:
+            print('{} is not found...try again!'.format(del_item))
+
+        # actually delete the item in list for all occurrences
+        for item in l:
+            if item == del_item:
+                l.remove(item)
+
+
+def series1():
+    """Return a series of action for 'series 1' assignment."""
+    # Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+    l = fruit_list()
+    # Display the list (plain old print() is fine…).
+    print(l)
+    # Ask the user for another fruit and add it to the end of the list.
+    l.append(add_fruit())
+    # Display the list.
+    print(l)
+    # Ask the user for a number and display the number back to the user and the
+    # fruit corresponding to that number (on a 1-is-first basis). Remember that
+    # Python uses zero-based indexing, so you will need to correct.
+    while True:
+        answer = int(input("Enter an integer between "
+                           "1 and {:d}!> ".format(len(l))))
+        if 1 <= answer <= 5:
+            break
+    print(answer, l[answer-1])
+    # Add another fruit to the beginning of the list using “+” and display
+    # the list.
+    l = [add_fruit()] + l
+    print(l)
+    # Add another fruit to the beginning of the list using insert() and display
+    # the list.
+    l.insert(0, add_fruit())
+    print(l)
+    # Display all the fruits that begin with “P”, using a for loop.
+    for item in l:
+        if item[0] == 'P':
```

Comment:
You could do `if item[0].lower() == 'p'` here in case the user entered 'pineapples' e.g.